base/docs/basedocs.jl

```diff
@@ -1447,8 +1447,20 @@ kw"struct"
     mutable struct
 
 `mutable struct` is similar to [`struct`](@ref), but additionally allows the
-fields of the type to be set after construction. See the manual section on
-[Composite Types](@ref) for more information.
+fields of the type to be set after construction.
+
+Individual fields of a mutable struct can be marked as `const` to make them immutable:
+
+```julia
+mutable struct Baz
+    a::Int
+    const b::Float64
+end
+```
```

Comment:
Only thing I don't know is whether you need an extra blank line after the block code. If this is rendered correctly as is, then it looks good to me.

Reply:
Doesn't seem to be the case. There are a few of these in the docs and they render properly